code/game/objects/structures/manned_turret.dm

```diff
@@ -38,33 +38,34 @@
 		buckled_mob.pixel_x = 0
 		buckled_mob.pixel_y = 0
 		if(buckled_mob.client)
-			buckled_mob.reset_perspective()
+			buckled_mob.client.change_view(world.view)

```

Comment:
because that's literally not its function and it resetting on reset_perspective would reset view changes on any forceMove